docs/pages/blog/turbo-1-11-0.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Turborepo 1.11
+date: 2023/11/27
+description: Turborepo 1.11 sets the Rust-based foundation for the future of Turborepo.
+tag: "web development"
+ogImage: /images/blog/turbo-1-11-0/x-card.png
+---
+
+# Turborepo 1.11
+
+import { Authors } from '../../components/Authors'
+import { ExperimentalBadge } from '../../components/ExperimentalBadge'
+import ThemeAwareImage from '../../components/ThemeAwareImage'
+import Date from "../../components/blog/Date";
+
+<Date>
+  Monday, November 27th, 2023
+</Date>
+
+<Authors authors={[
+  "gregsoltis",
+  "chrisolszewski",
+  "nicholasyang",
+  "alexanderlyon",
+]} />
+
+Turborepo 1.11 completes our migration to Rust and ships several developer experience improvements:
+
+- [**Our new Rust foundation**](#unlocking-the-future-of-turbo-with-rust): We've finished the port from Go to lay the groundwork for better performance, improved stability, and new features.
+- [**Group logs for improved readability**](#group-logs-for-improved-readability): You can now specify `--log-order=grouped` to organize your logs into distinct sections separated by tasks.
+- [**Updated examples**](#updated-examples): Start with an example for Next.js, Svelte, Remix, Nuxt, and more.
+
+Update today by running `npx @turbo/codemod migrate`.
+
+## Unlocking the future of `turbo` with Rust
+
+As we wrote about in [our blog post about the migration](https://vercel.com/blog/turborepo-migration-go-rust), we're excited to start seeing the benefits of our porting effort for Turborepo users. Rust has allowed us to surface more complexity in _our_ code so we're able to create a better experience in _your_ codebase in several ways:
+
+- We've noticed a slight performance improvement for the majority of use cases, even without doing much performance tuning in our code yet. We look forward to diving deeper into making `turbo` even faster soon.
```

Comment:
Should we add some metrics? Feels pretty vague here. How much, and which use cases? Even naming one or two would be helpful in my opinion.